recipes/vctk/hifigan/train_hifigan.py

```diff
@@ -0,0 +1,75 @@
+from trainer import Trainer, TrainerArgs
+
+from TTS.config.shared_configs import BaseAudioConfig
+from TTS.utils.audio import AudioProcessor
+from TTS.vocoder.configs import HifiganConfig
+from TTS.vocoder.datasets.preprocess import load_wav_data
+from TTS.vocoder.models.gan import GAN
+
+output_path = "/home/julian/workspace/train"
```

Comment:
Why it is pointin Julian's workspace?

Why there is HiFiGan recipe update in VITS PR?

Reply:
My bad, I removed these commits, it is in the PR: https://github.com/coqui-ai/TTS/pull/1526